app/globals.css

```
@@ -35,24 +35,26 @@
   --font-sans:
     var(--font-manrope), ui-sans-serif, system-ui, sans-serif, "Apple Color Emoji",
     "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  --font-display:
+    var(--font-source-serif), ui-serif, Georgia, "Times New Roman", serif;
```

Stylelint (17.14.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)